jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/SwitchesTests.java

```diff
@@ -0,0 +1,100 @@
+package io.github.epam.bootstrap.tests.composite.section;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.switches;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.CoreMatchers.is;
+
+/**
+ * Created by Dmitrii Pavlov on 08.10.2019
+ * Email: delnote@gmail.com; Skype: Dmitrii Pavlov
+ */
+
+public class SwitchesTests extends TestsInit {
+
+
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void isValidationTests() {
+        switches.checkbox
+                .is()
+                .hidden()
+                .enabled()
+                .core()
+                .attr("type", "checkbox")
+                .hasClass("custom-control-input")
+                .tag(is("input"));
+        switches.checkbox.label()
+                .is()
+                .displayed()
+                .enabled()
+                .core()
+                .hasClass("custom-control-label")
+                .text(is("Toggle this switch element"))
+                .tag(is("label"));
+        switches.checkboxContainer
+                .is()
+                .displayed()
+                .enabled()
+                .core()
+                .hasClass("custom-control custom-switch")
+                .tag(is("div"));
+    }
+
+    @Test
+    public void disabledIsValidationTests() {
+        switches.checkboxDisabled
+                .is()
+                .hidden()
+                .disabled()
+                .core()
+                .attr("type", "checkbox")
+                .hasClass("custom-control-input")
+                .tag(is("input"));
+        switches.checkboxDisabled.label()
+                .is()
+                .displayed()
+                .enabled()
+                .core()
+                .hasClass("custom-control-label")
+                .text(is("Disabled switch element"))
+                .tag(is("label"));
+        switches.checkboxDisabledContainer
+                .is()
+                .displayed()
+                .enabled()
+                .core()
+                .hasClass("custom-control custom-switch")
```

Comment:
to make more clear error if such error will be occured, we should to devide class names here like:
                 .hasClass("custom-control")
                 .hasClass("custom-switch")